packages/next-preact/readme.md

```diff
@@ -34,15 +34,12 @@ require('@zeit/next-preact/alias')()
 const { createServer } = require('http')
 const next = require('next')
 
-
 const app = next({ dev: process.env.NODE_ENV !== 'production' })
 const port = process.env.PORT || 3000
 const handle = app.getRequestHandler()
 
-app.prepare()
```

Comment:
These changes are due to prettier editing the file from my editor... I can revert it if you'd like.